astrokat/test/check-cals-select.sh

```diff
@@ -7,13 +7,13 @@ GREEN="\033[1;32m"
 YELLOW="\033[0;33m"
 NOCOLOR="\033[0m"
 
-CMD="astrokat-targets.py --cat-path ../../catalogues/"
+CMD="astrokat-cals.py --cat-path ../../catalogues/"
+date='2018-08-06 12:34'
```

Comment:
Not sure if this will/might override builtin bash commands, perhaps change the case to be safe.
```suggestion
DATE='2018-08-06 12:34'
```